cratedb_toolkit/io/processor/kinesis_lambda.py

```diff
@@ -123,8 +123,13 @@ def handler(event, context):
             logger.debug(f"Record Data: {record_data}")
 
             # Process record.
-            sql = cdc.to_sql(record_data)
-            connection.execute(sa.text(sql))
+            operation = cdc.to_sql(record_data)
+            connection.execute(sa.text(operation.statement), parameters=operation.parameters)
+
+            # Processing alternating CDC events requires write synchronization.
+            # FIXME: Needs proper table name quoting.
+            connection.execute(sa.text(f"REFRESH TABLE {CRATEDB_TABLE}"))
+
```

Comment:
Hereby, I am adding a REFRESH TABLE operation again. Please share your objections if you have any.

/cc @wierdvanderhaar, @hlcianfagna, @hammerhead, @proddata, @widmogrod